test/evergreen.yml

```diff
@@ -4568,6 +4570,22 @@ tasks:
                 cat $t && rm $t
             fi
 
+            # Generate the code complexity statistics and upload the analysis into Atlas charts
+            virtualenv -p python3 venv
+            source venv/bin/activate
+            pip3 install pandas
+
+            python3 "../metrixplusplus/metrix++.py" view --format=python > ../code_statistics_report/code_complexity_summary.json
+            python3 "../metrixplusplus/metrix++.py" export --db-file=metrixpp.db > ../code_statistics_report/metrixpp.csv
+
+            python3 ../test/evergreen/code_complexity_analysis.py -s ../code_statistics_report/code_complexity_summary.json -d ../code_statistics_report/metrixpp.csv -o ../code_statistics_report/atlas_out_code_complexity.json
+      - func: "upload stats to atlas"
+        vars:
+          stats_dir: ./wiredtiger/code_statistics_report
+          test-name: code_complexity
+          # Override the collection name to use the code complexity collection.
```

Comment:
I think we should remove the default setting for collection value at the code statistics builder level, and let each task take care of its own collection value setting. And thus remove the need for this comment line.

Reply:
Yes, I thought about it but the collection `CodeCoverage` is being used by 50+ functions so I implemented this way to use the `CodeCoverage` collection as a default setting and override it once in the `cyclomatic-complexity`